README.md

```diff
@@ -20,6 +20,41 @@ The `@microsoft/fast-tooling` package contains a web worker referred to as the M
 
 The `@microsoft/fast-tooling-react` package contains various React components that work with the message system provided by `@microsoft/fast-tooling` to edit data, render data as HTML, and navigate data. To learn more, check out the package [README](./packages/fast-tooling-react).
 
+### Publishing
```

Comment:
This entire section should be in the `CONTRIBUTING.md` file, the README is for visitors to the repository to get an idea of what this repository contains, and in fact you could consider creating a separate markdown file for devops, like `DEV_OPS.md` and cross linking it from the `CONTRIBUTING.md` file. We are doing this with our `STYLE_GUIDE.md`